src/screens/ExchangeModal.tsx

```diff
@@ -120,7 +133,7 @@ const getInputHeaderTitle = (type, defaultInputAsset) => {
   }
 };
 
-const getShowOutputField = type => {
+const getShowOutputField = (type: 'deposit' | 'swap' | 'withdrawal') => {
```

Comment:
@jxom Do we have a re-usable type for this?

Reply:
we have `ExchangeModalTypes`